### ISSUE TITLE
Please add support for staging channels

### ISSUE THREAD

**Issue by TuskarMA** [OutwalkStudios/discord-backup#23]
Loading backups with stage channels causes error

```
Job Failed: Invalid Form Body
user_limit[NUMBER_TYPE_MAX]: int value should be less than or equal to 99.
ID: loadChannel::guild.channels.create::Собрание
DiscordAPIError[50035]: Invalid Form Body
user_limit[NUMBER_TYPE_MAX]: int value should be less than or equal to 99.
    at handleErrors (C:\Users\V3land\Documents\backup_test\node_modules\@discordjs\rest\dist\index.js:722:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SequentialHandler.runRequest (C:\Users\V3land\Documents\backup_test\node_modules\@discordjs\rest\dist\index.js:1120:23)
    at async SequentialHandler.queueRequest (C:\Users\V3land\Documents\backup_test\node_modules\@discordjs\rest\dist\index.js:953:14)
    at async _REST.request (C:\Users\V3land\Documents\backup_test\node_modules\@discordjs\rest\dist\index.js:1266:22)
    at async GuildChannelManager.create (C:\Users\V3land\Documents\backup_test\node_modules\discord.js\src\managers\GuildChannelManager.js:171:18) {
  requestBody: {
    files: undefined,
    json: {
      name: 'Собрание',
      topic: undefined,
      type: 2,
    }
  },
  rawError: {
    message: 'Invalid Form Body',
    code: 50035,
    errors: { user_limit: [Object] }
  },
  code: 50035,
  status: 400,
  method: 'POST',
  url: 'https://discord.com/api/v10/guilds/1195297326730842182/channels'
}

```

In backup file there is a 
userLimit: 10000 value, which I think is default for stage channels

**Comment by jleeson** [comment]
Hey @TuskarMA, Would you mind sharing the stage channel backup data? The channel type in the stack trace appears to be a normal voice channel, If you can provide some backup data to check with that would be helpful in resolving this issue.

**Comment by jleeson** [comment]
Hey @TuskarMA, thanks for opening this issue. I just added support for Stage channels and released it in v0.7.0. Feel free to open another issue if you have any more problems!
